spring-cloud-dataflow-module-deployer-spi/src/main/java/org/springframework/cloud/dataflow/module/deployer/ModuleDeployer.java

```diff
@@ -38,6 +39,8 @@
 
 	public static final String JMX_DEFAULT_DOMAIN_KEY  = "spring.jmx.default-domain";
 
+	public static final String MODULE_DEPLOYMENT_GROUP_ID = "spring_cloud_dataflow.module_deployment_group_id";
```

Comment:
Make sense. I thought of it as the stream being a "module deployment group" as in "group of modules that got deployed". "Group deployment" works as well, I suppose, and I like "group deployment id" being similar with "module deployment id". Still, would want to keep the `dataflow.` prefix, correct?